lib/features/dind.js

```diff
@@ -76,7 +77,7 @@ export default class DockerInDocker {
     };
   }
 
-  killed(task) {
+  async killed(task) {
     task.runtime.gc.removeContainer(this.container.id);
```

Comment:
This is so the method is treated as a promise for all features so we can call .then() in states.js . Can't call then() on a regular method